test/ui/transfer-item.test.ts

```diff
@@ -44,9 +44,9 @@ describe("UI - Transfer Items", () => {
     game.move.select(MoveId.DRAGON_CLAW);
 
     game.onNextPrompt("SelectModifierPhase", UiMode.MODIFIER_SELECT, () => {
-      expect(game.scene.ui.getHandler()).toBeInstanceOf(ModifierSelectUiHandler);
+      expect(game.scene.ui.getCurrentHandler()).toBeInstanceOf(ModifierSelectUiHandler);
 
-      const handler = game.scene.ui.getHandler() as ModifierSelectUiHandler;
+      const handler = game.scene.ui.getCurrentHandler() as ModifierSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
      const handler = game.scene.ui.getCurrentHandler<ModifierSelectUiHandler>();
```